packages/starlight/components/SidebarSublist.astro

```diff
@@ -36,7 +35,14 @@ interface Props {
 						open={flattenSidebar(entry.entries).some((i) => i.isCurrent) || !entry.collapsed}
 					>
 						<summary>
-							<span class="large">{entry.label}</span>
+							<div class="group-label">
+								<span class="large">{entry.label}</span>
+								{entry.badge && (
+									<>
+										<Badge text={entry.badge.text} variant={entry.badge.variant} />
```

Comment:
And then we can also include it here. (Never done this whitespace hack in this kind of context before, but I think it should work — it’s required due to https://github.com/withastro/compiler/issues/852)

```suggestion
										{' '}
										<Badge text={entry.badge.text} variant={entry.badge.variant} />
```